app/models/hostgroup.rb

```diff
@@ -137,18 +137,22 @@ def inherited_lookup_value(key)
     [key.default_value, _("Default value")]
   end
 
-  # returns self and parent parameters as a hash
-  def parameters(include_source = false)
+  def parent_params(include_source = false)
     hash = {}
     ids = ancestor_ids
-    ids << id unless new_record? or self.frozen?
     # need to pull out the hostgroups to ensure they are sorted first,
     # otherwise we might be overwriting the hash in the wrong order.
-    groups = ids.size == 1 ? [self] : Hostgroup.sort_by_ancestry(Hostgroup.includes(:group_parameters).find(ids))
+    groups = Hostgroup.sort_by_ancestry(Hostgroup.includes(:group_parameters).find(ids))
     groups.each do |hg|
       hg.group_parameters.each {|p| hash[p.name] = include_source ? {:value => p.value, :source => N_('hostgroup').to_sym, :safe_value => p.safe_value, :source_name => hg.title} : p.value }
     end
+    hash
+  end
 
+  # returns self and parent parameters as a hash
+  def parameters(include_source = false)
```

Comment:
Right now hostgroup shows parent parameters and host shows all inherited global parameters so this returns a different result then `host_params` from Host.